### ISSUE TITLE
[Bug]: Custom Element doesn't exist: Mushroom-X-card

### ISSUE THREAD

**Issue by VincentDePincent** [piitaya/lovelace-mushroom#1589]
### Requirements

- [X] I checked the troubleshooting section in the README to verify that I have the latest Mushroom version.
- [X] I did a search to see if there is a similar issue or if a pull request is open.
- [X] I have read the state of card mod support (in pin issues) and understand card mod support is not provided in this repository.

### Current Behavior

I have my entire dashboard made up of mushroom cards. When i load it from the companion app, it shows the error mentioned in the title, like so
![Screenshot_20241116-101429](https://github.com/user-attachments/assets/2037b935-fd64-41b6-9c44-863bd7f5bca4)
This only happens if i use the companion app, my duckdns url and local IP work just fine.
Reinstalling works, but only once. When reloaded it happens again. Same goes for clearing cache.

### Expected Behavior

Simply displaying mushroom cards correctly. When accessed from local or url, it works fine:
![Screenshot_20241116-102029~2](https://github.com/user-attachments/assets/664f7308-63ca-49f8-919f-b3e9039c5860)
](url)

### Steps To Reproduce

Open the companion app
Switch from Overview to dashboard.
Wait a second, the dashboard will remain empty until the errors load.

### Context

YAML state
```yaml

```

### Environment

```markdown
- Browser: App
- HA Version: 24.11.1
- Mushroom: 4.1.1
```

**Comment by piitaya** [comment]
Have you try all these steps? It's a cache/resource issue, we can't do anything about the mushroom code to fix that issue.
.
https://github.com/piitaya/lovelace-mushroom?tab=readme-ov-file#troubleshooting

**Comment by tbowmo** [comment]
I have followed the guide to clear cache (removed mushroom resources, uninstalled mushroom in hacs, installed it in hacs again, reloaded HA, rebooted HA, cleared cache in browser, tried incognito mode in browser).

I cannot find any mushroom cards when adding a new card to my dashboard. The existing ones are gone..

**Comment by piitaya** [comment]
Does it work with an older version of Mushroom? What device and browser do you use?

**Comment by tbowmo** [comment]
@piitaya 4.1.0 works, 4.1.1 doesn't.
Browser is both chrome and app, both behaves similar.

**Comment by piitaya** [comment]
If you have browser logs to share it would be very useful as I can not reproduce it.

**Comment by tbowmo** [comment]
As requested, browser logs
![image](https://github.com/user-attachments/assets/f139b95c-0307-45f7-be47-39d7f43d2c24)

**Comment by tbowmo** [comment]
as requested
![image](https://github.com/user-attachments/assets/23c9e7e9-776c-497e-b3a4-2d2a8407d3df)

**Comment by VincentDePincent** [comment]
So, a day after i posted this, after weeks of trouble, mine seems to be consistently up and running again. I have no clue what changed, i didn't do anything, but i'm glad it works.

**Comment by piitaya** [comment]
> as requested ![image](https://private-user-images.githubusercontent.com/5305197/387575805-23c9e7e9-776c-497e-b3a4-2d2a8407d3df.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIwMzQ0NTEsIm5iZiI6MTczMjAzNDE1MSwicGF0aCI6Ii81MzA1MTk3LzM4NzU3NTgwNS0yM2M5ZTdlOS03NzZjLTQ5N2UtYjNhNC0yZDJhODQwN2QzZGYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTExOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMTlUMTYzNTUxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDg2NjA1NjliZDBkYzcxZDJlZjg0NzhmNzE2MTk3NDE0YWFiNzllZGRiMDA4ZGRlNGFhOWI1N2I4NzQ4YWEyMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.N7VQuzSGMkxBp91wVcjHnfh6Tnz7j4fw51Su2UJKaeg)

Looks like it's more a issue for loading ressource than a issue in Mushroom code, weird...

Can you share a screenshot of your ressources? https://my.home-assistant.io/redirect/lovelace_resources/
Also, do you have ressources declared in yaml?

**Comment by tbowmo** [comment]
Funny thing, it seems to only be when accessing the dashboard from the "outside" (ie. at work). If I'm at home, it works fine..
Below is the resources I load (both from yaml, and otherwise)

![image](https://github.com/user-attachments/assets/184679b8-0283-44a5-80ce-497b2eda2f7e)
![image](https://github.com/user-attachments/assets/fae69411-58b9-4587-b428-abbee5d7032e)

**Comment by VincentDePincent** [comment]
Yikes, i just updated to 4.2.0 and it came back. I get the exact same result as tbowmo.

**Comment by piitaya** [comment]
I really thing it's a cache issue on your side with the ressources. Most of the changes in 4.2.0 is translations based so there is not big technical change.

**Comment by smartathomeat** [comment]
https://github.com/piitaya/lovelace-mushroom/releases

**Comment by devbf** [comment]
I had the same problem. Somehow the following steps solved it for me:

1. Settings -> Dashboards -> Resources
2. Click on the mushroom.js row
3. Update
4. Now everything loads properly...

**Comment by devbf** [comment]
And on my Android wall tablet, I had to delete the cache of the Home Assistant Companion application to solve the issue.

**Comment by DIYtechie** [comment]
> hey tnere: SOLUTION for me was the following instructions: Hope to help u guys! ![Bildschirmfoto 2024-11-27 um 18 45 37](https://private-user-images.githubusercontent.com/100084082/390517218-2d08cd6d-9336-47cd-ac22-afd22fc9fc24.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY3NjIwODcsIm5iZiI6MTczNjc2MTc4NywicGF0aCI6Ii8xMDAwODQwODIvMzkwNTE3MjE4LTJkMDhjZDZkLTkzMzYtNDdjZC1hYzIyLWFmZDIyZmM5ZmMyNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExM1QwOTQ5NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZDA4ZDEwNWMyNWU2MjAxODMxOGI5YmZkZTg0ZmI0OGNlZjBlYzVkMjAxYzZhZTQyYjBiZWZlNDhjYzNiNmZkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ep0uYAk0BCKa8Hkot4A0eT0sr2EQApoLv8rp7DE78hE)

Thanks! This worked for me. The resources path for all my cards were /hacsfiles/ and needed changing to be changed /local/community/
